sawtooth-daml-rpc/src/main/java/com/blockchaintp/sawtooth/daml/rpc/SawtoothWriteService.java

```diff
@@ -264,7 +264,19 @@ private String makeDamlCommandDedupKeyAddress(final SubmitterInfo submitterInfo)
 
   private synchronized Future sendToValidator(final Batch batch) {
     // Push to TraceTransaction class
-    this.sawtoothTransactionsTracer.putWriteTransactions(batch.toString());
+    StringBuilder txnInJson = new StringBuilder();
```

Comment:
Don't do this use a com.google.protobuf.util.JsonFormat here, saves lines, way less fragile

Reply:
This has been replaced with JsonFormat